openmrs-keycloak-smart-auth/src/main/java/org/openmrs/contrib/keycloak/smart/auth/provider/SmartLaunchAuthenticator.java

```diff
@@ -82,6 +82,7 @@ public void authenticate(AuthenticationFlowContext context) {
 
 		final String launchContext = context.getAuthenticationSession().getClientNote(LAUNCH_CLIENT_REQUEST_PARAM);
 		if (!StringUtils.isEmpty(launchContext)) {
+			context.getAuthenticationSession().setUserSessionNote(SMART_NOTE_PREFIX+"patient",launchContext);
```

Comment:
This should be using the same constant as above, but for now:

```suggestion
			context.getAuthenticationSession().setUserSessionNote(SMART_NOTE_PREFIX + "patient", launchContext);
```